contrib/seeds/README.md

```diff
@@ -10,8 +10,9 @@ to addrman with).
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
 
-    curl -s http://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
+    curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
```

Comment:
Maybe rename "Onion" to "Torv2"?

Reply:
> should the deps section be modified for the Ubuntu issue you saw?

Nah. I'm sure it's fixed in Ubuntu 22.04, which I'll be using everywhere by next major release. Could mention it as a temporary workaround, but I decided not to for a seldom-used dev script like this.

Reply:
> Maybe rename the "Onion" column to "Torv2" in the tool output?

it's actually TorV3 now. I changed the regexp. That's why none are returned. It's optimistically aimed at the crawler indexing TorV3's some day, or otherwise, it doesn't hurt.

Reply:
> it's actually TorV3 now

Ah yes, length 16 -> 56.

> I'm sure it's fixed in Ubuntu 22.04, which I'll be using everywhere by next major release. 

Sounds good.